### PR TITLE
More support to read from Config

### PR DESCRIPTION
**Pull request by ennru**

### CODE REVIEW

File: src/main/scala/pl/project13/kaze/KazeClass.scala [hunk 1]
```diff
@@ -467,6 +485,9 @@ object KazeClass {
     }
 
     private def up(s: String): String = s.charAt(0).toUpper + s.tail
+
+    private def hyphenize(s: String): String =
+      s.charAt(0).toLower + s.tail.replaceAll("""([A-Z])""", "\\-$1").toLowerCase(Locale.ENGLISH)
```
Comment (patriknw) [inline src/main/scala/pl/project13/kaze/KazeClass.scala]: In Akka we use `s.toLowerCase(Locale.ROOT)`

Comment (ktoso) [reply src/main/scala/pl/project13/kaze/KazeClass.scala]: Ping?

Comment (ennru) [reply src/main/scala/pl/project13/kaze/KazeClass.scala]: This code is not part of the generated stuff, changed it anyhow.

**Comment by ktoso** [comment]
Merged, purely administrative stuff I can do AFAIK. Not done on work time.

**Pull request merged**